@here/harp-mapview/lib/MapViewFog.ts

```diff
@@ -159,17 +159,18 @@ export class MapViewFog {
         this.m_scene.traverse(object => {
             if (object instanceof THREE.Mesh) {
                 if (object.material instanceof THREE.Material) {
-                    if (
-                        object.material instanceof THREE.Material &&
-                        // HighPrecisionLineMaterial does not support fog
-                        !(object.material instanceof HighPrecisionLineMaterial)
-                    ) {
-                        if (object.material instanceof SolidLineMaterial) {
-                            const material = object.material;
-                            (material as SolidLineMaterial).updateFog(enableFog);
+                    // HighPrecisionLineMaterial does not support fog
```

Comment:
MINOR: The deeply nested if clauses could be refactored with guard clauses.
e.g.
```javascript
if (!(object instanceof THREE.Mesh)) {
    return;
}
if (!(object.material instanceof THREE.Material)) {
    return;
}
// ...
```

Reply:
> MINOR: The deeply nested if clauses could be refactored with guard clauses.
> e.g.
> 
> ```js
> if (!(object instanceof THREE.Mesh)) {
>     return;
> }
> if (!(object.material instanceof THREE.Material)) {
>     return;
> }
> // ...
> ```

Good point! Done.